scripts/build.ps1

```diff
@@ -805,16 +815,19 @@ function Create-NugetPackages
     $tpNuspecDir = Join-Path $env:TP_PACKAGE_PROJ_DIR "nuspec"
 
     # Copy over the nuspecs to the staging directory
-    $nuspecFiles = @("TestPlatform.TranslationLayer.nuspec",
-                     "TestPlatform.ObjectModel.nuspec",
-                     "TestPlatform.TestHost.nuspec",
-                     "TestPlatform.CLI.nuspec",
-                     "TestPlatform.Build.nuspec",
-                     "TestPlatform.Extensions.TrxLogger.nuspec", 
-                     "Microsoft.NET.Test.Sdk.nuspec",
-                     "Microsoft.TestPlatform.nuspec",
-                     "Microsoft.TestPlatform.Portable.nuspec",
-                     "Microsoft.CodeCoverage.nuspec")
+    $nuspecFiles = @(
+        "TestPlatform.TranslationLayer.nuspec",
+        "TestPlatform.ObjectModel.nuspec",
+        "Microsoft.TestPlatform.AdapterUtilities.nuspec",
```

Comment:
nit nit nit: could we order them on name?

Reply:
done.